src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -812,11 +812,7 @@ void M<T>(T t)
 }
 ";
             var c = CreateCompilation(new[] { source });
-            c.VerifyDiagnostics(
-                // (6,10): warning CS8629: The suppression operator (!) should be used in code within a '#nullable' context.
```

Comment:
Something to think about, from an IDE perspective, is generating a hidden diagnostic for these "useless" suppressions so that they can easily generate quickfixes to turn on the feature or remove unnecessary suppressions.